aliyun/src/main/java/org/apache/iceberg/aliyun/oss/OSSInputFile.java

```diff
@@ -29,11 +30,19 @@
   private Long length = null;
 
   OSSInputFile(OSS client, OSSURI uri) {
-    super(client, uri);
+    this(client, uri, new AliyunProperties());
```

Comment:
Is it necessary to have these constructors that default properties? That seems like a bad thing to me. If they're just for testing, then can we pass properties in the other tests?

Reply:
done